frontend/src/App.jsx

```diff
@@ -1,19 +1,23 @@
 import React from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-
+import { StylesProvider } from "@material-ui/core/styles";
 import Home from "./components/home";
+import SignUpForm from "./components/UserSignUp/SignUpForm";
 
 function App() {
   return (
-    <Router>
-      <div>
-        <Switch>
-          <Route exact path="/">
-            <Home />
-          </Route>
-        </Switch>
-      </div>
-    </Router>
+    <StylesProvider injectFirst>
+      <Router>
+        <div>
+          <Switch>
+            <Route exact path="/">
+              <Home />
+              <SignUpForm />
```

Comment:
Could you move SignUpForm into a new route called "/registration"?

Reply:
Yep!